common/src/sql/dbinit.sql

```diff
@@ -1419,6 +1419,83 @@ CREATE INDEX ON omicron.public.update_available_artifact (
     targets_role_version
 );
 
+/*
+ * System updates
+ */
+CREATE TABLE omicron.public.system_update (
+    /* Identity metadata (asset) */
+    id UUID PRIMARY KEY,
+    time_created TIMESTAMPTZ NOT NULL,
+    time_modified TIMESTAMPTZ NOT NULL,
```

Comment:
What does it mean for an update to be modified? The version updated?

Reply:
Also, is there a need for an index to be created for this or other tables here?

Reply:
I don't really know why these would be modified. The only reason I have it on here is because it's convenient to make these `Asset`s, which are like `Resource` but without name and description. I don't think it's too bad to have the field there and never use it — the alternative (not being an `Asset`) is probably worse, though I haven't thought about it too hard.

Yes to indexes, will put that on my to-do list.